src/gui/app.cpp

```diff
@@ -13,6 +13,65 @@
 
 namespace NeovimQt {
 
+struct ConnectorInitArgs {
+	enum Type {
+		Embed,
+		Server,
+		Spawn,
+		Default
+	};
+
+	const Type type;
+	const int timeout;
+	const QString server;
+	const QString nvim;
+	const QStringList positionalArgs;
+	const QStringList neovimArgs;
+};
+
+ConnectorInitArgs::Type getConnectorType(const QCommandLineParser &parser)
```

Comment:
nitpick: `CommandLineParser& parser`

Reply:
Is there a clang-format or some other format configuration file I can use? I didn't know what the style was so just ended up using the default clang-format abs adapting it a little bit so it looks  similar to the rest of the code. I missed this one.